src/worker/exporters/node_exporter.py

```diff
@@ -106,6 +210,19 @@ def init_config(job_id, port=None):
             elif field_name == 'net_tx':
                 config['counter'][field_name] = shell_cmd(args, 5).split()[9]
 
+        elif 'cpu' in field_name:
+            if 'util' in field_name:
+                config['fieldFiles'][field_name] = '/proc/stat'
+
+                # Call cpu_percent to get intial 0.0 values
+                _ = psutil.cpu_percent(percpu=True)
+
+            elif 'frequency' in field_name:
+                config['fieldFiles'][field_name] = '/proc/cpuinfo'
```

Comment:
Are  '/proc/cpuinfo' and '/proc/meminfo' being used? if not lets remove them from here.

Reply:
Addressed